test/nsidc-smap/test_configuraton.py

```diff
@@ -0,0 +1,214 @@
+from harmony import BBox, Collection
+import copy
+
+non_production_configuration = {
+    "single_output_tests": {
+        "subset_bounding_box": {
+            "SPL2SMA": {
+                "request_params": {
+                    "collection": Collection(id="C1242631429-NSIDC_CUAT"),
+                    "granule_id": "G1260377350-NSIDC_CUAT",
+                    "spatial": BBox(-8.2, 42.0, 12.7, 52.8),
+                    "variables": ["Radar_Data/sigma0_hh_mean"],
+                    "format": "image/tiff",
+                    "labels": ["smap-rtest-1", "smap-rtests"],
+                },
+                "test_params": {"ext": ".tif"},
+            },
+            "SPL3FTP_E": {
+                "request_params": {
+                    "collection": Collection(id="C1263071066-NSIDC_CUAT"),
+                    "granule_id": "G1263078049-NSIDC_CUAT",
+                    "spatial": BBox(-26, 63, -12, 67),
+                    "variables": [
+                        "Freeze_Thaw_Retrieval_Data_Polar/freeze_thaw",
+                        "Freeze_Thaw_Retrieval_Data_Polar/latitude",
+                        "Freeze_Thaw_Retrieval_Data_Polar/longitude",
+                    ],
+                    "labels": ["smap-rtest-15", "smap-rtests"],
```

Comment:
These test numbers refer to the original test ID from the spreadsheet.  I could renumber if you wanted, but thought this was easier to decipher.